src/pug/includes/body.pug

```diff
@@ -18,7 +18,7 @@ include ./fork.pug
                         label.sr-only(for="filter") Filter opportunity difficulty
                         select#filter
                             for filter in ['All difficulties', 'Easy', 'Medium', 'Hard']
```

Comment:
We can just update the array value rather than adding the string splitting logic